packages/shared/src/contexts/PushNotificationContext.tsx

```diff
@@ -126,6 +128,22 @@ export function PushNotificationContextProvider({
     [],
   );
 
+  const { completeAction, checkHasCompleted, isActionsFetched } = useActions();
+  const isPushEnabled = isPushSupported && isSuccess && isEnabled;
+  const hasCompletedAction =
+    isActionsFetched &&
+    isPushEnabled &&
+    isSubscribed &&
+    checkHasCompleted(ActionType.EnableNotification);
```

Comment:
It will never fire right, since `checkHasCompleted` will always be false until you call `completeAction`?